src/contact-summary/contact-summary-emitter.js

```diff
@@ -32,7 +32,7 @@ function emitter(contactSummary, contact, reports) {
       throw new Error("You cannot set appliesToType to an array which includes the type 'report' and another type.");
     }
     
-    if (appliesToType.includes('report')) {
+    if (appliesToType.includes('report') || (appliesToType.length === 0 && typeof card.fields === 'function')) {
```

Comment:
I think `"report"` is a special case. It doesn't make sense to have a appliesToType of "report" and "person" and line 32 will throw an error on this case.

The original issue was mostly concerned about matching any contact, so if you revert this line it will fall through to line 48 which you've modified to add the card assuming it's related to a contact.